tests/orm/models/test_models.py

```diff
@@ -138,6 +143,16 @@ def required_arguments(request, default_user, aiida_localhost, tmp_path):
         return StructureData, {'cell': [[1, 0, 0], [0, 1, 0], [0, 0, 1]]}
     if request.param is RemoteData:
         return RemoteData, {'remote_path': '/some/path', 'computer': aiida_localhost}
+    if request.param is RemoteStashData:
+        return RemoteStashData, {'stash_mode': StashMode.COMPRESS_TAR}
```

Comment:
```suggestion
        return RemoteStashData, {'stash_mode': StashMode.COMPRESS_TAR.value}
```
Has to be a str, not a member of enum. Why even the tests are passing for this? :smiling_face_with_tear: